addons/sourcemod/scripting/ConnectAnnounce.sp

```diff
@@ -989,12 +1091,83 @@ public void Announcer(int client, int iRank, bool sendToAll)
 #if defined _Connect_Included
```

Comment:
A better way to handle this would be to replace the value with either the one retrieved from the external plugin or else replace with an empty string to handle a single connect message template withtout having to do if conditions in it